x-pack/plugins/security_solution/common/detection_engine/transform_actions.ts

```diff
@@ -48,7 +48,7 @@ export const transformAlertToRuleAction = ({
   id,
   params,
   action_type_id: actionTypeId,
-  ...(alertsFilter && { alerts_filter: alertsFilter }),
+  // ...(alertsFilter && { alerts_filter: alertsFilter }),
```

Comment:
Let's not forget to uncomment this.

Reply:
let me remove that, I am pretty sure that's why we have some test failling